### DISCUSSION TITLE
Should we create a schema for suppliers?

### DISCUSSION THREAD

**Discussion by italvi** [Festo-se/cyclonedx-editor-validator#238] [Ideas]
Should we create a separate schema for suppliers of SBOMs, as they e.g. don't have to follow everything that could only be required for internal components/products.

**Comment by mmarseu** [comment]
Do you have something in mind where the supplier schema would differ from the current "custom" one?

**Reply by italvi** [reply]
E.g. copyright Festo is not required.
